### PR TITLE
Conditional statements handling inputs + covariate file support in phewas.R

### PR DESCRIPTION
**Pull request by evagradovich**
# This PR does the following:

- Adds conditional statements around input data
- Implements covariate file support 

# Details
- Adds conditional statements towards the top of `main.nf` ensuring only one set of genotype data is supplied.
- Implements covariate file support in `phewas.R` script for phewas call and in `main.nf`
- Fixes GL contig removal in vcf->plink conversion

# Test
On Lifebit CloudOS:
[link](https://cloudos.lifebit.ai/public/jobs/611285377db707019a04c46c)

### CODE REVIEW

File: bin/phewas.R [hunk 1]
```diff
@@ -105,9 +109,15 @@ results_d = data.frame()
 write.csv(results_d, file=paste0(outprefix,"_phewas_results.csv"), row.names=FALSE)
 #write.csv(results_d, file=paste0(outprefix,"_top_results.csv"), row.names=FALSE)
```
Comment (mmeier93) [inline bin/phewas.R:110]: If this isn't remove, just remove it 👍

File: bin/phewas.R [hunk 2]
```diff
@@ -105,9 +109,15 @@ results_d = data.frame()
...
 }
+print(covariate_file)
```
Comment (mmeier93) [inline bin/phewas.R:112]: Can be removed as well 👍

**Review by mmeier93 — APPROVED** [review]
Looking good 👍 

A few things:
- See comments
- Renaming the 2 of the params to make it clear it clear that they are lists
- A few changes are needed for the container
  - Make sure default container is `container='quay.io/lifebitai/phewas:latest'`
  - But leave the non-quay for now for the coloc processe(s) + make an issue about this 
  - Make an issue to remember to add optional Rmd blocks like so (for coloc) - like so: https://github.com/lifebit-ai/gel-pre-ingestion-qc/blob/b50e43929b59dadefab5831abe52b15091b316be/bin/report.Rmd#L322

**Comment by evagradovich** [comment]
All comments addressed

**Pull request merged**